src/Identity/Models/RegisterResponseModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Bit.Core.Models.Api;
+
+namespace Bit.Identity.Models;
+
+public class RegisterResponseModel : ResponseModel
```

Comment:
This is fine for now, but we should probably extract this to a `CaptchaProtectedResponse` if we actually create more captcha protected endpoints.

Reply:
I'm going to go ahead and do this

Reply:
I used an interface instead of a base model, but let me know if you'd like to see it handled differently.

Reply:
that's perfect, thanks